### ISSUE TITLE
YELL and SAY chat channels identified as UNKNOWN on CHAT_MSG_ADDON

### ISSUE THREAD

**Issue by LenweSaralonde** [Stanzilla/WoWUIBugs#24]
Messages sent via SendAddonMessage() for the newely implemented YELL and SAY channels are identified as UNKNOWN when received by the CHAT_MSG_ADDON event.

Reproduction steps:
```lua
/script C_ChatInfo.RegisterAddonMessagePrefix("BlzBug"); 
/script local f = CreateFrame("Frame"); f:RegisterEvent("CHAT_MSG_ADDON"); f:SetScript("OnEvent", function(self, event, prefix, text, channel) print("Received", event, prefix, string.len(text), channel, text) end);

/script C_ChatInfo.SendAddonMessage("BlzBug", "Hello world!", "WHISPER", UnitName("player")) -- OK: WHISPER channel
/script C_ChatInfo.SendAddonMessage("BlzBug", "Hello world!", "YELL") -- KO: UNKNOWN channel
/script C_ChatInfo.SendAddonMessage("BlzBug", "Hello world!", "SAY") -- KO: UNKNOWN channel

**Comment by Meorawr** [comment]
This might have been fixed since 1.13.4 if anyone's willing to confirm (at least one person so far has said YELL is correct).

**Comment by Meorawr** [comment]
![image](https://user-images.githubusercontent.com/287102/82840231-c2151c80-9ec9-11ea-926e-456b51098da0.png)
YELL is also confirmed fixed (but lacking screenshot).
